core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4070,7 +4084,7 @@ public synchronized HttpRedirect doQuietDown() {
      *
      * @param block Block until the system really quiets down and no builds are running
      * @param timeout If non-zero, only block up to the specified number of milliseconds
-     * @deprecated since 2.267; use {@link #doQuietDown(boolean, int, String)} instead.
+     * @deprecated since 2.267; use {@link #doQuietDown2(boolean, int, String, boolean)} instead.
```

Comment:
Traditionally we haven't updated existing deprecation notices, because people love scavenger hunts :trollface: (kidding, great to see you take care of this!)